### PR TITLE
[docs] Add example for how to import the library

### PR DESCRIPTION
**Pull request by BewareMyPower**

### CODE REVIEW

**Comment by BewareMyPower** [comment]
There is a fix needed in the upstream. When `CMAKE_BUILD_TYPE` is not defined, the release library and debug library could be the same.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ For the supported Pulsar features, see [Client Feature Matrix](https://pulsar.ap
...
 
+## Import the library into your project
+
+### CMake with vcpkg integration
+
+Navigate to [vcpkg-example](./vcpkg-example) for how to import the `pulsar-client-cpp` into your project via vcpkg.
+
+### Download pre-built binaries
+
+For non-vcpkg projects, you can download pre-built binaries from the [official release page](https://pulsar.apache.org/download/#pulsar-c-client).
```
Comment (shibd) [inline README.md:38]: Maybe we also need to update this doc? https://pulsar.apache.org/docs/3.1.x/client-libraries-cpp/

Comment (BewareMyPower) [reply README.md:38]: Yes.

**Review by shibd — APPROVED** [review]

**Comment by BewareMyPower** [comment]
@RobertIndie Have you fetched the submodule? Please note that this PR also updates the commit of the vcpkg submodule. Could you run the following command and show the output?

```bash
cd vcpkg && git log --pretty=oneline | head -n 1
```

**Review by RobertIndie — APPROVED** [review]

**Comment by RobertIndie** [comment]
> @RobertIndie Have you fetched the submodule? Please note that this PR also updates the commit of the vcpkg submodule. Could you run the following command and show the output?
> 
> ```shell
> cd vcpkg && git log --pretty=oneline | head -n 1
> ```
> 
> If you only ran `git apply 379.diff`, maybe it won't work.

Thanks. It works now!

**Pull request merged**